ci/Makefile

```diff
@@ -11,16 +11,26 @@ ifeq ($(HERBERT_BRANCH),)
 	@exit 1
 endif
 
+.PHONY: check_inventory
+check_inventory:
+ifeq ($(ANSIBLE_INVENTORY),)
+	@echo ""
+	@echo "Must set ANSIBLE_INVENTORY. Use \"ansible/inventory\" to use invetory"
+  @echo	"from current repository."
+	@echo ""
+	@exit 1
+endif
+
```

Comment:
Question: Should we default `ANSIBLE_INVENTORY` to `ansible/inventory` if not set? Or is it better to force the user to always set this var?

If we want to default it we would just add the following to the top of the Makefile

```Makefile
ANSIBLE_INVENTORY ?= ansible/inventory
```

Reply:
Uh huh. Likewise add "HERBERT_BRANCH ?= local" or should the branch be required?

Reply:
Sure I think that's a good default too!